Feat: Card and hero image snippet were added

DanielGovrin opened the pull request.

Commit:
creat team card 2 board

Commit:
Create hero title board

Commit:
removed 2 boards

Commit:
create hero image

Commit:
fix the tsx

Commit:
create team cards 1 board

Commit:
fix tsx code

Commit:
add mq to Hero title snippet

Commit:
add mq to Hero Image

Commit:
add mq to teamcards 1

Commit:
add mq to team card 2

Commit:
remove services board

Commit:
fix image

Commit:
fix mq for Hero Image

Commit:
fix mq for Hero title

Commit:
fix mq for Team card 1

Commit:
fix mq for Team cards 2

Commit:
changed root to snippet name

Commit:
fix: renamed css classed and removed unecessary rules

Commit:
mq inside selector - fix

Commit:
mq inside of selector

Commit:
fix: consistency in mq width

Commit:
fix: changed board name

Commit:
refactor: Remove unnecessary CSS file

alisey commented:
What's the context for this? It doesn't follow the guidelines we've set for snippets. The boards that served as examples `hero-center.board.tsx` and `hero-flex.board.tsx` deleted with no explanation. `services-section.board.tsx` is deleted, but `services-section.board.css` is left.

DanielGovrin commented:
Apologies, I misunderstood something that Dana told me. I’ll add those files back to the repo.

Commit:
revert: Restore previously removed files

Restored hero-center.board.tsx/css, hero-flex.board.tsx,  services-section.board.tsx after miscommunication about deletion.

Commit:
fix: add missing newline at end of board files

Dana-Levy commented:
hero-center.board.tsx
hero-flex.board.tsx 

These aren't the final snippets

0xRezaa commented on _codux/boards/services-section.board.tsx:
```diff
@@ -76,4 +76,4 @@ export default createBoard({
         windowHeight: 650,
     },
-});
+});
```
please add the end of line break

0xRezaa commented:
Just a recording of me reviewing the hero-image visually

https://github.com/user-attachments/assets/c9a97264-6482-4a72-8f34-c24df6d625ae

0xRezaa commented on _codux/boards/card-collection-1.board.tsx:
```diff
@@ -0,0 +1,70 @@
+                    <div className="card-image-box">
+                        <img
+                            src="https://static.wixstatic.com/media/8586f3_f6d2538ac6374f71b6dc29511c5cd1a3~mv2.png/v1/fill/w_860,h_600/Frame%206.png"
+                            alt="Delicious Margherita Pizza"
```
I think that having an alt value is a benefit as we insentivise the users to set them, maybe we want to use a different value than this.

0xRezaa commented:
Just a recording of me reviewing the hero-title visually

https://github.com/user-attachments/assets/d3d972b1-b499-4331-92d1-7401c6914b69

0xRezaa commented:
Just a recording of me reviewing the card-collection-1 visually


https://github.com/user-attachments/assets/bd3ebd54-987c-48ca-90ab-0e40aa7955ed

0xRezaa commented:
Just a recording of me reviewing the card-collection-2 visually

https://github.com/user-attachments/assets/5e67cac4-09aa-448e-aaa8-ea29ac0e7f4f

0xRezaa commented on _codux/boards/card-collection-2.board.css:
```diff
@@ -0,0 +1,89 @@
+    min-height: 800px;
+    max-height: 700px;
+
+    @media (width<= 1024px) {
```
Is this supposed to be `width < 1024px`

See the recording

https://github.com/user-attachments/assets/8115cb32-20da-415a-bfcf-a67081bc34c9

Commit:
fix: fixes according to cr
